tictactoe/tictactoe.css

```diff
@@ -0,0 +1,76 @@
+article {
+    margin      : 0;
+    line-height : 0.5;
+}
+
+#next {
```

Comment:
Of course, you'll need to change your accessors based on your HTML `id` changes.